src/Payments.Core/Domain/Invoice.cs

```diff
@@ -107,11 +107,16 @@ public string GetFormattedId()
         [DisplayFormat(DataFormatString = "{0:C}")]
         public decimal Subtotal { get; private set; }
 
+        [DisplayFormat(DataFormatString = "{0:C}")]
+        [DisplayName("Taxable Amount")]
+        public decimal TaxableAmount { get; set; }
+
         [DisplayFormat(DataFormatString = "{0:C}")]
         [DisplayName("Tax")]
         public decimal TaxAmount { get; private set; }
 
         [DisplayFormat(DataFormatString = "{0:C}")]
+        [DisplayName("Tax Rate")]

```

Comment:
I'm changing this is my test, but please confirm you want Total to be called Tax Rate. If yes, dismiss this review.